msbuild/tests/Xamarin.iOS.Tasks.Tests/ProjectsTests/EmbeddedExtension.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+using Xamarin.Tests;
+
+namespace Xamarin.iOS.Tasks {
+	[TestFixture ("iPhone")]
+	[TestFixture ("iPhoneSimulator")]
+	public class EmbeddedExtension : ProjectTest {
+		
+		public EmbeddedExtension (string platform) : base (platform)      
+		{
+		}
+
+		[Test]
+		public void BasicTest ()
+		{
+			var mtouchPaths = SetupProjectPaths ("ManagedContainer", "../NativeExtensionEmbedding/managed", true, Platform);
+
+			Engine.ProjectCollection.SetGlobalProperty ("Platform", Platform);
+
+			var xcodeProjectFolder = Path.Combine (mtouchPaths.ProjectPath , "..", "..", "native");
+			string [] xcodeBuildArgs;
+			if (this.Platform  == "iPhone")
+				xcodeBuildArgs = new [] { "-configuration", "Debug", "-target", "NativeTodayExtension", "-arch", "arm64", "-sdk", "iphoneos13.5" };
+			else
+				xcodeBuildArgs = new [] { "-configuration", "Debug", "-target", "NativeTodayExtension", "-arch", "x86_64", "-sdk", "iphonesimulator13.5" };
```

Comment:
won't asking for `13.5` makes it depend on Xcode 11.5 ?

Reply:
Good point - let me see if I can drop that param.

Reply:
Yep, I can. Removed.